containers/jekyll/README.md

```diff
@@ -6,20 +6,46 @@
 
 | Metadata | Value |  
 |----------|-------|
-| *Contributors* | [Carlos Mendible](https://github.com/cmendible) |
+| *Contributors* | [Carlos Mendible](https://github.com/cmendible), [parkr](https://github.com/parkr) |
 | *Categories* | Community, Languages, Frameworks |
 | *Definition type* | Dockerfile |
+| *Published images* | mcr.microsoft.com/vscode/devcontainers/jekyll |
+| *Published image architecture(s)* | x86-64 |
 | *Works in Codespaces* | Yes |
 | *Container host OS support* | Linux, macOS, Windows |
 | *Container OS* | Debian |
 | *Languages, platforms* | Ruby, Jekyll |
 
+See **[history](history)** for information on the contents of published images.
+
+## Using this definition
+
 In addition to Ruby and Bundler, this development container installs Jekyll and the required tools at startup:
 
 - If your Jekyll project contains a `Gemfile` in the root folder, the development container will install all gems at startup by running `bundle install`. This is the [recommended](https://jekyllrb.com/docs/step-by-step/10-deployment/#gemfile) approach as it allows you to specify the exact Jekyll version your project requires and list all additional Jekyll plugins.
 - If there's no `Gemfile`, the development container will install Jekyll automatically, picking the latest version. You might need to manually install the other dependencies your project relies on, including all relevant Jekyll plugins.
 
-## Using this definition
+You can decide how often you want updates by referencing a [semantic version](https://semver.org/) of each image. For example:
+
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0`
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0.0`
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0.0.1`
+
+See [history](history) for information on the contents of each version and [here for a complete list of available tags](https://mcr.microsoft.com/v2/vscode/devcontainers/ruby/tags/list).
+
+Alternatively, you can use the contents of `base.Dockerfile` to fully customize your container's contents or to build it for a container host architecture not supported by the image.
+
+### Installing Node.js
+
+Given JavaScript front-end web client code written for use in conjunction with a PHP back-end often requires the use of Node.js-based utilities to build, this container also includes `nvm` so that you can easily install Node.js. You can change the version of Node.js installed or disable its installation by updating the `args` property in `.devcontainer/devcontainer.json`.
```

Comment:
```suggestion
Given JavaScript front-end web client code written for use in conjunction with a Jekyll site often requires the use of Node.js-based utilities to build, this container also includes `nvm` so that you can easily install Node.js. You can change the version of Node.js installed or disable its installation by updating the `args` property in `.devcontainer/devcontainer.json`.
```